src/js/components/DataTable/Header.js

```diff
@@ -183,6 +191,15 @@ const Header = ({
               }
             } else content = header;
 
+            if (unitsContent) {
+              content = (
+                <Box direction="row" align="baseline" gap="xsmall">
```

Comment:
Good point. I'll add the align/gap to the header.units part of the theme. Is there a good place to put defaults other than adding them to the base grommet theme or is that the place?

The reason it's baseline is otherwise it's 'center' which doesn't work well for something with 2 different font sizes like this.

Reply:
>Is there a good place to put defaults other than adding them to the base grommet theme or is that the place?

Since the styling of the unit is retrieved from the theme I think it will be the best place.